### PR TITLE
Appstream metainfo

### LINKED ISSUE #3492: Add AppStream Metadata file for Linux
### Describe the issue
In order for package managers to gain more meta info (screenshots, bug tracker, donation URL, ...), applications should provide an [AppStream Metadata file](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html).

### Reproduction steps
No template in https://github.com/opengisch/QField/tree/master/platform/linux

### Expected behavior
Application should provide AppStream Metadata file for Linux.

### Observed behavior
No AppStream Metadata file provided for Linux.

### Desktop (please complete the following information)
 - OS: Linux
 - QGIS Version
 - QFieldSync Version

### Mobile (please complete the following information)
 - Device:
 - OS:
 - QField version:

### PR DESCRIPTION
**Pull request by darkdragon-001**
Fixes #3492

### CODE REVIEW

**Comment by m-kuhn** [comment]
Thank you for opening this PR :heart: 

```
-- Deploying desktop files -- 
ERROR: No such file or directory:  /home/runner/builddir/_CPack_Packages/Linux/External/AppImage/usr/share/qfield/qfield.desktop 
CMake Error at /home/runner/builddir/CPackExternal.cmake:8 (execute_process):
  execute_process failed command indexes:

    1: "Child return code: 1"
```

Looks like

**Comment by darkdragon-001** [comment]
Damned, I forget the include (even though I added it in the issue) :see_no_evil:

**Comment by m-kuhn** [comment]
@darkdragon-001 ignore the failing test please, the app image has been produced and can be downloaded here: https://github.com/opengisch/QField/actions/runs/3316636101

**Comment by darkdragon-001** [comment]
@m-kuhn I checked the app image and the file is in the correct location :tada:

**Comment by m-kuhn** [comment]
Let me check how I can start the pipelines with the proper permissions to produce signed apks

File: platform/linux/qfield.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	<description>
+		<p>QField focuses on efficiently getting GIS fieldwork done and exchanging data between the field and the office in a comfortable and user friendly way.
+
+Built on top of the popular QGIS open source project, QField lets users consume fully configured projects in the field allowing for customized feature forms, map themes, print layouts, and more, bringing the power of QGIS at your fingertips.
+</p>
+	</description>
+	<launchable type="desktop-id">qfield.desktop</launchable>
+	<url type="homepage">https://qfield.org/</url>
+	<url type="bugtracker">https://github.com/opengisch/QField/issues</url>
+	<url type="donation">https://github.com/opengisch</url>
```
Comment (nirvn) [inline platform/linux/qfield.metainfo.xml:17]:
```suggestion
	<url type="donation">https://github.com/opengisch</url>
	<url type="help">https://docs.qfield.org/</url>
```

**Comment by m-kuhn** [comment]
Seems to be not so easy to elevate privileges ... :thinking:

**Comment by m-kuhn** [comment]
@nirvn should we merge and deal with followup problems as they appear?

**Review by nirvn — APPROVED** [review]

**Pull request merged**